### PR TITLE
Add building for different targets

### PR DESCRIPTION
**Pull request by DaniilKl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,54 @@ something similar to (the exact tasks numbers may differ):
   NOTE: Tasks Summary: Attempted 2532 tasks of which 2524 didn't need to be rerun and all succeeded.
   ```
 
+### Building for different targets
+
+Note, that Zarhus OS is supported by several targets. You can find supported
+targets in every `conf/machine` directories for every BSP layer.
+
+To change the target you are building your image for, you can edit the `machine`
```
Comment (macpijan) [inline README.md]: Editing a file and building from `-dirty` tree is a suboptimal approach.

Can we instead specify target as a command line option? Either override the default value from .yml file, or remove the target from the .yml file and instruct in the docs that it should be always provided via cmd line?

Comment (DaniilKl) [reply README.md]: I will check that.

Comment (m-iwanicki) [reply README.md]: We can override at least `KAS_DISTRO`, `KAS_MACHINE`, `KAS_TARGET`, `KAS_TASK`.
Overriding machine variable was mentioned in this PR here: https://github.com/zarhus/meta-zarhus/blob/c34c57099a0a9abb026f8376cbcff99dfc79ebbf/README.md?plain=1#L98

Comment (DaniilKl) [reply README.md]: Addressed [here](https://github.com/zarhus/meta-zarhus/compare/c34c57099a0a9abb026f8376cbcff99dfc79ebbf..f247a584d112ff6335f21347f85fc838b7bb34a8)

Comment (m-iwanicki) [reply README.md]: Why did you remove `zarhus-machine-cm3.conf`?

Comment (DaniilKl) [reply README.md]: Because it is surplus and unused.

Comment (DaniilKl) [reply README.md]: But I have forgotten to remove it from recipes :man_facepalming:

Comment (m-iwanicki) [reply README.md]: Then remove all mentions of this target?

```
$ grep -R zarhus-machine-cm3
meta-zarhus-bsp/meta-zarhus-bsp-rockchip/recipes-kernel/linux/linux-yocto_%.bbappend:COMPATIBLE_MACHINE:zarhus-machine-cm3 = "zarhus-machine-cm3"
README.md:  $ cd build/tmp/deploy/images/zarhus-machine-cm3
README.md:  $ sudo bmaptool copy zarhus-base-image-zarhus-machine-cm3.wic.gz /dev/sdx
README.md:  zarhus-base-image-zarhus-machine-cm3.wic.bmap zarhus-base-image-zarhus-machine-cm3.wic.gz /dev/sdx
README.md:  bmaptool: info: copying image 'zarhus-base-image-zarhus-machine-cm3.wic.gz' to block device '/dev/sdx' using bmap file 'zarhus-base-image-zarhus-machine-cm3.wic.bmap'
```

Comment (DaniilKl) [reply README.md]: Yep. fixing it...

Comment (DaniilKl) [reply README.md]: Removed all references [here](https://github.com/zarhus/meta-zarhus/compare/67cd47afbe5a9127d56b562c0af3823e6ecefd30..3916ceed897d08ecfb838f8aa0f338c618b63fa3).

Comment (DaniilKl) [reply README.md]: Added to `zarhus/docs` here https://github.com/zarhus/docs/pull/11.

**Review by macpijan — APPROVED** [review]
LGTM, feel free to merge when rdy

**Pull request merged**